backend/src/main/java/com/google/backend/servlets/AlertConfigurationServlet.java

```diff
@@ -102,4 +112,25 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     return parameters;
   }
+
+  public final class Configuration {
+    private final String user;
+    private final String dimension;
+    private final String metric;
+    private final String relatedDimension;
+    private final String relatedMetric;
+
+    private Configuration(String user, String dimension, String metric, String relatedDimension, String relatedMetric) {
+      this.user = user;
+      this.dimension = dimension;
+      this.metric = metric;
+      this.relatedDimension = relatedDimension;
+      this.relatedMetric = relatedMetric;
+    }
+
```

Comment:
optional: you can add a method
```
public static Configuration createFromEntity(Entity entity) {
  return new Configuration(
    (String) entity.getProperty(USER_PROPERTY),
    ...
  );
}
```
Then when you want to convert an `Entity` to a `Configuration` object you can do `Configuration configuration = Configuration.createFromEntity(entity);` in line 55.